src/js/shield_defs.js

```diff
@@ -340,19 +296,18 @@ function pentagonUpShield(
   radius1 = radius1 ?? 2;
   radius2 = radius2 ?? 0;
   return {
-    backgroundDraw: (ref) =>
-      ShieldDraw.pentagon(
-        true,
-        offset,
-        angleInRadians,
-        fillColor,
-        strokeColor,
-        ref,
-        radius1,
-        radius2,
-        1,
-        rectWidth
-      ),
+    canvasDrawnBlank: {
+      drawFunc: "pentagon",
+      params: {
```

Comment:
Set `pointUp` to `true`, or else this shield will be upside-down.

Reply:
I'm going to change the default for pentagon instead, since all of our pentagons point up at this point.  We can still set `pointUp=false` if we need to override that behavior.

Reply:
Will that affect home plate shields, which use the same draw function but still point downward?